src/main/java/com/direwolf20/buildinggadgets/common/items/gadgets/GadgetDestruction.java

```diff
@@ -72,34 +86,30 @@ private int getCostMultiplier(ItemStack tool) {
     public void addInformation(ItemStack stack, @Nullable World world, List<ITextComponent> tooltip, ITooltipFlag flag) {
         super.addInformation(stack, world, tooltip, flag);
         tooltip.add(TooltipTranslation.GADGET_DESTROYWARNING
-                            .componentTranslation()
-                            .setStyle(Styles.RED));
+                .componentTranslation()
```

Comment:
![](https://media1.tenor.com/images/5a9f96460bdee4161996a9b616fc1ca7/tenor.gif?itemid=12272188)

Had to.

Reply:
This is intended as a cleanup PR, so formatting should not be a problem. Also is that a 40-space indentation originally?

Reply:
Propably... IntelliJ formatted it like that so that it stayed below the part the chainging came from...
If you ask me: how it is now looks strange to me. But I don't really care, so If you want to keep it like that, just do it.